signalfx-tracing/signalfx-dotnet-tracing/aspnetcore-and-mongodb/docker-compose.yml

```diff
@@ -27,8 +29,23 @@ services:
     restart: unless-stopped
     ports:
       - "27017:27017"
-  smart-gateway:
-    image: smart-gateway
-    restart: unless-stopped
+  otel-collector:
+    image: otel/opentelemetry-collector-contrib:0.45.0
```

Comment:
I think we should rather use `quay.io/signalfx/splunk-otel-collector:0.45.0`

It is our recommendation and also should simplify this example (the OOTB config should be good).

Here is a docker-compose example: https://github.com/signalfx/splunk-otel-collector/tree/main/examples/docker-compose

Reply:
I will give it a try later today...